pom.xml

```diff
@@ -89,19 +89,11 @@
 			</dependency>
 
 			<!-- End OpenMRS core -->
-			
 			<dependency>
-				<groupId>org.mockito</groupId>
-				<artifactId>mockito-all</artifactId>
-				<version>1.8.5</version>
-				<scope>test</scope>
-			</dependency>
-
-			<dependency>
-				<groupId>org.codehaus.groovy</groupId>
+				<groupId>org.apache.groovy</groupId>
 				<artifactId>groovy-all</artifactId>
 				<version>${groovyVersion}</version>
-				<type>jar</type>
```

Comment:
The same goes for changing jar to pom

Reply:
https://mvnrepository.com/artifact/org.codehaus.groovy/groovy-all/3.0.24